### PR TITLE
src/config/graphical-session/wayland.md: Add the river compositor

### PR DESCRIPTION
**Pull request by Osrepnay**
Add the [river](https://github.com/riverwm/river) compositor to the list of packaged compositors

### CODE REVIEW

File: src/config/graphical-session/wayland.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Void Linux currently packages the following Wayland compositors:
 - Wayfire: 3D Wayland compositor
 - Hikari: a stacking compositor with some tiling features
 - Cage: a Wayland kiosk
+- River: a dynamic tiling Wayland compositor
```
Comment (ericonr) [inline src/config/graphical-session/wayland.md:29]: I appreciate the change, but this makes me think we should consider removing the list entirely.

File: src/config/graphical-session/wayland.md [hunk 2]
```diff
@@ -51,9 +52,8 @@ installing the `qt5-wayland` package and setting the environment variable
 applications also require installing the `kwayland` package.
 [EFL](https://wayland.freedesktop.org/efl.html)-based applications require
 setting the environment variable `ELM_DISPLAY=wl`, and can have issues without
-it, due to not supporting XWayland properly.
-[SDL](https://libsdl.org)-based applications require setting the environment
-variable `SDL_VIDEODRIVER=wayland`.
+it, due to not supporting XWayland properly. [SDL](https://libsdl.org)-based
+applications require setting the environment variable `SDL_VIDEODRIVER=wayland`.
```
Comment (ericonr) [inline src/config/graphical-session/wayland.md:56]: Please split into a separate commit.

Comment (Osrepnay) [reply src/config/graphical-session/wayland.md:56]: Unless this should be in a separate pull request, fixed
